src/components/DataTable/Cells/clients.js

```diff
@@ -2,6 +2,7 @@ const clientName = {
   id: 'clientName',
   Header: 'Client',
   accessor: client => client.clientName,
+  filterMethod: ({ value }, { clientName }) => clientName.toLowerCase().includes(value.toLowerCase()),
```

Comment:
Seeing a lot of repetition here, again some technical debt to make note of just.